lib/reverse_string.rb

```diff
@@ -1,5 +1,21 @@
 def reverse_characters(str)
+#Ok this is a little better but im still not sure how to use the .join to get it back to a string
+#The array can be called back backwards but its not "saved" anywhere in that order so it cant be converted to a string
+string="Today is a good day"
```

Comment:
This method is meant to be generic–it should work for **any** string at all. And that hypothetical string is already represented by `str` (the "parameter" of this method). So you can remove `string="Today is a good day"`–just imagine you have a string already, and it's stored (already) in a variable named `str`.